src/pages/migrating/faq.mdx

```diff
@@ -123,3 +133,193 @@ description:
 
 _We'll have more on this once we have interviewed teams that have successfully
 migrated to v11._
+
+## Sass and package management
+
+### What packages should I be installing?
+
+If you're using our React components, `@carbon/react` will have everything you
+need to use along with it: component styles, icons, grid, theming and type. This
+package essentially includes all our most used code assets in Carbon. If you've
+installed this package, you should not have to install our styles, grid, theme,
+icons or type packages.
+
+In order to use our styles with this package, all you need to include is the
+following in your `index.scss`:
+
+```scss
+@use '@carbon/react';
+```
+
+We understand that this can be a bit bulky, and sometimes you don't want all of
+the styles that come with that. To include styles for something more specific,
+we suggest including them like so:
+
+```scss
+@use '@carbon/react/scss/components/[componentname]';
+```
+
+| Carbon package   | What is included                                                                                |
+| ---------------- | ----------------------------------------------------------------------------------------------- |
+| `@carbon/react`  | React components, Carbon component sass and helper mixins, color themes, typeface, Carbon grids |
+| `@carbon/styles` | Carbon component sass and helper mixins                                                         |
+| `@carbon/grid`   | Carbon CSS and flex grids                                                                       |
+| `@carbon/themes` | white, g10, g90, g100 themes                                                                    |
+| `@carbon/colors` | design language color tokens                                                                    |
+| `@carbon/type`   | IBM Plex fonts                                                                                  |
```

Comment:
I think we should remove this to avoid implying that users should install these alongside one another.